src/lottery/assembly/fee-strategies.ts

```diff
@@ -1,36 +1,36 @@
 import { u128 } from "near-sdk-as";
 
 export enum StrategyType {
-  constant = 0 as u8,
```

Comment:
capitalize enum names and use standard values unless explicitly needed (e.g. values are defined outside of code/context and needs to be explicitly fixed to it)